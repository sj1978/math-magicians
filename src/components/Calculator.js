import React from 'react';

import Buttons from './Buttons';

const Calculator = () => (

  <section>

    <Buttons />

  </section>
);
export default Calculator;
