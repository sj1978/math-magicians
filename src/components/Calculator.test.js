import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from './Calculator';

describe('Calculator', () => {
  test('renders Calc component', () => {
    const tree = renderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
