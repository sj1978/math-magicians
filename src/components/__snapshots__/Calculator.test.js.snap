// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders Calc component 1`] = `
<section
  className="calculator"
>
  <h3
    className="calc-title"
  >
    Lets do some math
  </h3>
  <section
    className="calc"
  >
    <div
      className="container"
    >
      <div
        className="display"
      >
        <p
          className="display-2"
        >
          0
        </p>
      </div>
      <div
        className="all-buttons"
      >
        <button
          className="btn all-clear"
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          className="btn last-clear"
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          className="btn "
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="btn operation"
          onClick={[Function]}
          type="button"
        >
          /
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="btn operation"
          onClick={[Function]}
          type="button"
        >
          X
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="btn operation"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="btn operation"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
        <button
          className="btn number zero"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          className="btn number"
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="btn equal operation"
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </section>
</section>
`;
